freetextresponse/tests.py

```diff
@@ -260,6 +261,25 @@ def test_compute_score(self, credit):
             {'value': credit.value, 'max_value': Credit.full.value},
         )
 
+    @ddt.data(Credit.zero)
```

Comment:
Remove ddt entirely from this function.  We only need ddt when we want to run the same test with different parameters. 

In this case, we only need to use one credit value so...

return_value=credit   <---- Just set return_value to Credit.zero.  

Don't forget to remove credit from the function args.